native/packages/react-native-bpk-component-button/BpkButton.podspec

```diff
@@ -0,0 +1,15 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name             = package['name']
+  s.version          = package['version']
+  s.summary          = package['description']
+  s.license          = package['license']
+  s.homepage         = 'https://github.com/skyscanner/backpack'
+  s.authors          = package['author']
+  s.source           = { :git => 'https://github.com/skyscanner/backpack.git', :tag => "v#{s.version}" }
+  s.source_files     = 'src/*[^\-test].js'
+  s.dependency         'React'
+end
```

Comment:
@k0nserv does this look ok to you? I looked at some stuff online and had a go at making a podspec.

Reply:
Also, seeing as we're not supporting Android yet, maybe we can leave the gradle part for now - what do you think?